apis/common/v1/resource.go

```diff
@@ -205,15 +205,17 @@ type ResourceSpec struct {
 	// THIS IS AN ALPHA FIELD. Do not use it in production. It is not honored
 	// unless the relevant Crossplane feature flag is enabled, and may be
 	// changed or removed without notice.
-	// ManagementPolicy specifies the level of control Crossplane has over the
-	// managed external resource.
+	// ManagementPolicy specifies the array of actions Crossplane is allowed to
+	// take on the managed and external resources.
 	// This field is planned to replace the DeletionPolicy field in a future
 	// release. Currently, both could be set independently and non-default
-	// values would be honored if the feature flag is enabled.
+	// values would be honored if the feature flag is enabled. If both are
+	// custom, the DeletionPolicy field will be ignored.
 	// See the design doc for more information: https://github.com/crossplane/crossplane/blob/499895a25d1a1a0ba1604944ef98ac7a1a71f197/design/design-doc-observe-only-resources.md?plain=1#L223
+	// and this one: https://github.com/crossplane/crossplane/blob/444267e84783136daa93568b364a5f01228cacbe/design/one-pager-ignore-changes.md
 	// +optional
-	// +kubebuilder:default=FullControl
-	ManagementPolicy ManagementPolicy `json:"managementPolicy,omitempty"`
+	// +kubebuilder:default={"*"}
+	ManagementPolicy ManagementPolicy `json:"managementPolicies,omitempty"`
```

Comment:
Not being consistent with the json tag here is bugging me. Should we?

```suggestion
	ManagementPolicies ManagementPolicies `json:"managementPolicies,omitempty"`
```

Reply:
Agreed, changed it